tests/taskcluster/bin/raptor_graph

```diff
@@ -250,15 +259,15 @@ getEmulator()
       // when not a PR 'base' should be BASE -1 commit (before the
       // merge) and 'patch' should be BASE_REV (has the merged code)
       if (process.env.GITHUB_PULL_REQUEST === '0' && config[raptorTask].build === "base") {
-        var app = config[raptorTask].params.raptorApps;
-        suiteName = app[0].toUpperCase() + app.slice(1) + " app launch test (on base -1 commit)";
-        config[raptorTask].params.raptorSuiteName = suiteName;
+        var oldSuiteName = config[raptorTask].params.raptorSuiteName;
+        var newSuiteName = oldSuiteName.substring(0, oldSuiteName.indexOf('test')) + "test (on base -1 commit)";
+        config[raptorTask].params.raptorSuiteName = newSuiteName;
         config[raptorTask].params.gaiaRev = gaiaBaseMinusOne;
       }
       if (process.env.GITHUB_PULL_REQUEST === '0' && config[raptorTask].build === "patch") {
-        var app = config[raptorTask].params.raptorApps;
-        suiteName = app[0].toUpperCase() + app.slice(1) + " app launch test (on base)";
-        config[raptorTask].params.raptorSuiteName = suiteName;
+        var oldSuiteName = config[raptorTask].params.raptorSuiteName;
+        var newSuiteName = oldSuiteName.substring(0, oldSuiteName.indexOf('test')) + "test (on base)";
```

Comment:
I think the only caveat here is that the TH name must never contain  the word "test" in it.